Move VM network docs (#62619)
openshift/openshift-docs: virt/vm_networking/virt-configuring-dpdk-workloads.adoc

:_content-type: ASSEMBLY
[id="virt-configuring-dpdk-workloads"]

> = Configuring Data Plane Development Kit workloads

Reviewer: ```suggestion
= Configuring cluster and VMs for DPDK workloads
```

Reviewer: I have not heard anybody using the full name

Author: We have to spell out abbreviations in full. But if you think it is sufficiently well-known, then I can use the full name in the first sentence and the abbreviation in the title.

include::_attributes/common-attributes.adoc[]
:context: virt-configuring-dpdk-workloads

toc::[]

> You can configure Data Plane Development Kit (DPDK) workloads for a cluster, project, or virtual machine to improve performance over SR-IOV networks.

Reviewer: ```suggestion
You can run Data Plane Development Kit (DPDK) workloads in VMs connected to SR-IOV networks.
```
